Twitch/Twitch.Base/Services/NewAPI/ScheduleService.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Newtonsoft.Json;
+using StreamingClient.Base.Util;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Twitch.Base.Models.NewAPI;
+using Twitch.Base.Models.NewAPI.Schedule;
+using Twitch.Base.Models.NewAPI.Users;
+
+namespace Twitch.Base.Services.NewAPI
+{
+    /// <summary>
+    /// APIs for schedule-based services
+    /// </summary>
+    public class ScheduleService : NewTwitchAPIServiceBase
```

Comment:
Can you add a Unit Test class for this as well? Just copy-paste the format from one of the other unit test classes.

Reply:
will do